hutch_python/tests/test_happi.py

```diff
@@ -63,3 +63,40 @@ def test_get_lightpath():
     # Check that we created a valid BeamPath with no inactive objects
     assert obj.name == 'TST'
     assert len(obj.devices) == 3
+
+
+# run test_happi_objs() without the excluded devices
+@conftest.requires_lightpath
+def test_happi_objs_without_exclude_devices():
+    logger.debug("test_happi_objs")
+    db = os.path.join(os.path.abspath(os.path.dirname(__file__)),
+                      'happi_db.json')
+    # patch lightpath configs to include test db beamline
+    conftest.beamlines['TST'] = ['X0']
+    conftest.sources.append('X0')
+    # Only select active objects
+    lc = get_lightpath(db, 'tst')
+    # in DeviceLoadLevel.STANDARD you search happi and look for devices with same beamline name
+    objs = get_happi_objs(db, lc, 'tst', DeviceLoadLevel.STANDARD)
+    assert len(objs) == 4
+
+
+# run test_happi_objs() with exclude_devices
+@conftest.requires_lightpath
+def test_happi_objs_with_exclude_devices():
+    logger.debug("test_happi_objs")
+    db = os.path.join(os.path.abspath(os.path.dirname(__file__)),
+                      'happi_db.json')
+    # patch lightpath configs to include test db beamline
+    conftest.beamlines['TST'] = ['X0']
+    conftest.sources.append('X0')
```

Comment:
That linked conftest is super messy, we should probably clean it up (and do this setup in an autouse fixture instead of on conftest import, among other things)

I bet the test this was copied from and the conftest configurations were written in different calendar years, we can probably clean up the test suite a bit. It doesn't have to be in this PR though.

Reply:
I removed `conftest.beamlines['TST'] = ['X0']` and `conftest.sources.append('X0')` from test_happi.py.